coreos-base/coreos/coreos-0.0.1.ebuild

```diff
@@ -100,6 +86,7 @@ RDEPEND="${RDEPEND}
 	)"
 
 RDEPEND="${RDEPEND}
+	amd64? ( app-admin/etcd-wrapper )
```

Comment:
Yes, we want etcd-wrapper on arm64.  We should get an arm64 quay.io image in the next etcd release.
See https://quay.io/repository/coreos/etcd?tab=tags.